lang/syn/src/parser/accounts/constraints.rs

```diff
@@ -332,11 +350,14 @@ pub struct ConstraintGroupBuilder<'ty> {
     pub address: Option<Context<ConstraintAddress>>,
     pub token_mint: Option<Context<ConstraintTokenMint>>,
     pub token_authority: Option<Context<ConstraintTokenAuthority>>,
+    pub token_token_program: Option<Context<ConstraintTokenProgram>>,
     pub associated_token_mint: Option<Context<ConstraintTokenMint>>,
     pub associated_token_authority: Option<Context<ConstraintTokenAuthority>>,
+    pub associated_token_program: Option<Context<ConstraintTokenProgram>>,
```

Comment:
Same here, consistency with `token_token_program` above and to make it more clear.

```suggestion
    pub associated_token_token_program: Option<Context<ConstraintTokenProgram>>,
```

Reply:
Sounds good